server/anthropic/anthropic.go

```diff
@@ -282,15 +310,12 @@ func (a *Anthropic) ChatCompletion(request llm.CompletionRequest, opts ...llm.La
 	}
 
 	go func() {
-		defer close(output)
-		defer close(errChan)
+		defer close(eventStream)
 
-		if err := a.streamChatWithTools(initialState); err != nil {
-			errChan <- err
-		}
+		_ = a.streamChatWithTools(initialState)
```

Comment:
I think we can remove the return here for this function entirely, because it is returning the errors through the stream. The only problem is there is one error that is not being returned through the channel. I don't know if that is an error or is intentional, but in any case the error is not managed. Other option would be extract here the error here, and send to the channel here.

Reply:
Good catch.